Civi/Api4/Log.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4;
+
+/**
+ * Log
+ *
+ * @since 5.45
+ * @package Civi\Api4
+ */
+class Log extends Generic\DAOEntity {
+
```

Comment:
This might be appropriate for this entity (might fix tests too):

```suggestion
  use Generic\Traits\ReadOnly;

```
@monishdeb